development.rst

```diff
@@ -23,15 +23,12 @@ If you're not using `Pipenv`_, to install `pre-commit`, run:
 Automated Testing
 -----------------
 
-All pull requests and merges are tested in `Travis CI <https://travis-ci.org/>`_
-based on the ``.travis.yml`` file.
+All pull requests and merges are tested in `GitHub Actions <https://github.com/pytest-dev/pytest-html/actions>`_
+which are defined inside ``.github`` folder.
 
-Usually, a link to your specific travis build appears in pull requests, but if
-not, you can find it on the
-`pull requests page <https://travis-ci.org/pytest-dev/pytest-html/pull_requests>`_
-
-The only way to trigger Travis CI to run again for a pull request, is to submit
-another change to the pull branch.
+To retrigger CI to run again for a pull request, you can use either use
+dropdown option, close and reopen pull-request or to just update the branch
```

Comment:
nitpick: 'use the dropdown option, close and reopen the pull-request or just update the branch'